deploy/crds/workspaces.ecd.eclipse.org_devworkspaces_crd.yaml

```diff
@@ -547,6 +547,26 @@ spec:
                         type: object
                     type: object
                   type: array
+                events:
+                  description: Bindings of commands to events
+                  properties:
+                    postStart:
```

Comment:
The description for each phase would be exrimily useful

Reply:
fixed